vars/main.yml

```
@@ -0,0 +1,2 @@
+# This variable is true if the host OS is linux
+ansible_os_is_linux: "{{ ansible_os_family in ['Debian', 'ArchLinux', 'RedHat'] }}"
```

```suggestion
ansible_os_is_linux: "{{ ansible_os_family in ['Debian', 'Archlinux', 'RedHat'] }}"
```
According to https://docs.ansible.com/ansible/latest/user_guide/playbooks_conditionals.html#ansible-facts-os-family `Archlinux` is the correct value (and also according to my PC :wink: ).